### PR TITLE
docs(k8sprocessor): describe configuration in yaml

### PR DESCRIPTION
**Pull request by astencel-sumo**
I believe this is much more readable.
There are still things to improve in this description.

### CODE REVIEW

File: pkg/processor/k8sprocessor/README.md [hunk 1]
```diff
@@ -9,87 +9,155 @@ in a cluster, keeps a record of their IP addresses and interesting metadata. Upo
...
+        jobName: k8s.job.name
+        namespaceName: k8s.namespace.name
+        nodeName: k8s.node.name
+        podID: k8s.pod.id
+        podName: k8s.pod.name
+        replicaSetName: k8s.replicaset.name
+        serviceName: k8s.service.name
+        statefulSetName: k8s.statefulset.name
+        startTime: k8s.pod.startTime
+
```
Comment (sumo-drosiek) [inline pkg/processor/k8sprocessor/README.md:100]: btw, we should use `resourceprocessor` to handle this feature

**Comment by sumo-drosiek** [comment]
I don't see association configuration

File: pkg/processor/k8sprocessor/README.md (hunk 1 above)
Comment (astencel-sumo) [reply pkg/processor/k8sprocessor/README.md:100]: 💯 agree

**Comment by astencel-sumo** [comment]
> I don't see association configuration

Because it wasn't there before, I will add it as a separate pull request.

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**